### ISSUE TITLE
add falsy arbitrary

### ISSUE THREAD

**Issue by larrybotha** [dubzzz/fast-check#624]
## 🚀 Feature Request

Add an arbitrary that allows one to evaluate against all falsy values, i.e. `false, 0, "", null, NaN, undefined`

## Motivation

This makes it quick and easy to evaluate a function against the range of all possible falsy values a function may receive, without having to remember them.

I often find myself writing a quick arbitrary for this, but I have to wonder... "did I forget something", which is what's great about property-based testing - it does the remembering for you.

JsVerify has a `falsy` arbitrary: https://github.com/jsverify/jsverify/commit/3782a12bc0c9beb66fc57acfb3d7403e3e923a2b#diff-aed6998294505b549318ccc8fb7389fc

## Example

```javascript
fc.assert(fc.property(fc.falsy(), value => {
	expect(() => myFunc(value)).not.toThrow()
}))
```

**Comment by dubzzz** [comment]
Great suggestion. Let's go for `fc.falsy`.

If you want to do it, feel free to open a PR

**Comment by larrybotha** [comment]
Will do so this week :)

**Comment by tinydylan** [comment]
Do we want a `fc.truthy` as well?

**Comment by dubzzz** [comment]
Not convinced of truthy

IMO truthy is just `fc.anything().filter(v => !!v)` so I prefer keeping it outside of fast-check to limit the number of primitives offered by the library to the ones that are complex to build or error prone. In addition, I'd would receive requests to add truthyString, truthyInteger or thuthyUnicodeString...

**Comment by dubzzz** [comment]
Meanwhile there is still an opened issue with `fc.falsy`: it does not generate `0n`. We need to discuss and add a flag for that. See associated PR

**Comment by dubzzz** [comment]
Feature released in version 1.26.0 of fast-check.

Please feel free to re-open the issue.
